libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -196,7 +196,10 @@ bool ASTJsonConverter::visit(IfStatement const& _node)
 
 bool ASTJsonConverter::visit(WhileStatement const& _node)
 {
-	addJsonNode(_node, "WhileStatement", {}, true);
+	addJsonNode(_node,
```

Comment:
Sorry for the style break.  Did I miss the style guide somewhere?  Or do you follow a well-known one by convention?